timepiece/crm/views.py

```diff
@@ -3084,3 +3085,36 @@ def project_download_attachment(request, project_id, attachment_id):
         return HttpResponseRedirect(project_attachment.get_download_url())
     except:
         return HttpResponse('Project attachment could not be found.')
+
+def get_minding(request):
+    user = request.user
+    projects = Project.objects.filter(
+        point_person=user, status=4)[:3]
+    projects_json = []
+    for project in projects:
+        projects_json.append({
+            'code': project.code,
+            'name': project.name,
+            'business': project.business.name,
+            'business_short': project.business.short_name,
+            'id': project.id,
+        })
+
+    return JsonResponse(projects_json, safe=False)
+
+def get_recent(request):
+    user = request.user
+    entries = Entry.objects.order_by('project__id', '-start_time')
+    projects = entries.values('project').distinct('project')[0:3]
+    projects_json = []
+    for uid in projects:
```

Comment:
Change this so that there are not `n` database calls (I know it is only three, but still).  Do this.  `for project in Project.objects.get(id__in=projects):`  Then you get drop the `Project.objects.get` below).